Apps/WebClient/src/ClientApp/src/components/errorCard.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div>
+        <b-alert show="show" variant="danger" dismissible class="no-print">
+            <h4>{{ getCodeText() }} {{ title }}</h4>
+            <span>{{ description }}</span>
+        </b-alert>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { Component, Prop } from "vue-property-decorator";
+
+@Component
+export default class ErrorCardComponent extends Vue {
+    @Prop() title: string = "";
+    @Prop() description: string = "";
+    @Prop() code: string = "";
+    @Prop() show: boolean = false;
+
+    private getCodeText(): string {
+        return !this.code ? "" : this.code + ": ";
```

Comment:
per @FuriousLlama , should we flip this code as follow, as it looks more natural :-)
`return this.code ? this.code + ": " : "";`